packages/vscode-extension/tests/specs/api/versionSpecificLinting.spec.ts

```diff
@@ -48,7 +57,7 @@ suite('Neo4j version specific linting spec', () => {
             new vscode.Position(5, 7),
             new vscode.Position(5, 12),
           ),
-          "The query used a deprecated function. ('id' has been replaced by 'elementId or consider using an application-generated id')",
```

Comment:
This change is needed because now that the linter is actively downloaded, the message included in the linter for the 2025.05 version used in the tests was slightly different to the one the default linter (for the current neo4j) includes